lib: add flag to drop connection when running in cluster mode (#54927)
nodejs/node: doc/api/net.md

* {integer}

> When the number of connections reaches the threshold of `server.maxConnections`.
> 
> 1. Node.js will close the connection if the process is not running in cluster mode.
> 2. Node.js will send the connection to other worker process by default if the process
>    is running in cluster mode, you can set `server.dropMaxConnection` to `true` to
>    close the connection.

Reviewer: ```suggestion
When the number of connections reaches the `server.maxConnections` threshold:

1. If the process is not running in cluster mode, Node.js will close the connection.

2. If the process is running in cluster mode, Node.js will, by default, route the connection to another worker process. To close the connection instead, set [`server.dropMaxConnection`][] to `true`.
```

It is not recommended to use this option once a socket has been sent to a child
with [`child_process.fork()`][].

### `server.dropMaxConnection`

<!-- YAML
added: REPLACEME
-->

* {boolean}

> Set this property to `true` to close the connection when the number of connections
> reaches the threshold of `server.maxConnections`. It only works in cluster mode.

Reviewer: ```suggestion
Set this property to `true` to begin closing connections once the number of connections reaches the [`server.maxConnections`][] threshold. This setting is only effective in cluster mode.
```

Author: Thanks. Done !
